admin/partials/settings/scheduler.php

```diff
@@ -45,6 +45,29 @@
 
 $scheduler_select = sprintf( '<select name="scheduler" id="scheduler">%1$s</select>', $scheduler_options );
 
+$intervals = array(
+	'*/5 * * * *'  => esc_html__( 'Every 5 Minutes', 'boldgrid-backup' ),
+	'*/10 * * * *' => esc_html__( 'Every 10 Minutes', 'boldgrid-backup' ),
+	'*/30 * * * *' => esc_html__( 'Every 30 Minutes', 'boldgrid-backup' ),
+	'0 * * * *'    => esc_html__( 'Once Every Hour', 'boldgrid-backup' ),
```

Comment:
Once a day may be too much... If I create a backup and a job was going to upload it to google, I'd want the job to run sooner than up to 24 hours later.

Why? I can't really put my finger on it... Why not have once a week?

My personal preference would be to have at most once every hour.